deps/fdasrsf/optimum_reparam.pyx

```diff
@@ -52,7 +53,7 @@ def coptimum_reparam_n(np.ndarray[double, ndim=1, mode="c"] mq,
     sizes = np.zeros(N, dtype=np.int32)
     Go = np.zeros((M, N))
     To = np.zeros((M, N))
-    for k in xrange(0, N):
```

Comment:
How did that work for you, when you changed the language level to 3?

Reply:
I suppose that xrange it is supported for cython in python 3. I changed it to be consistent with the python3 style (It's just speculation).